src/OpenTelemetry.Extensions.Propagators/README.md

```diff
@@ -0,0 +1,47 @@
+# Trace context propagators for OpenTelemetry .NET
```

Comment:
Title is not accurate...

Reply:
I have now called that `Propagator formats for OpenTelemetry .NET` - as it is likely to include other formats in the future such as Jaeger. Does it reflect the purpose better?